include/Cylinder.hpp

```diff
@@ -0,0 +1,268 @@
+#ifndef CYLINDER_HPP
+#define CYLINDER_HPP
+
+#include "NavierStokes.hpp"
+
+// In this file we define the classes for the flow past a cylinder test cases.
+// To do this, we create a template class for the cylinder test case, which
+// contains the common features of the test cases. This class is a template on
+// the dimension, because it needs to be defined as constexpr for technical
+// reasons. Then, we create two derived classes, one for the 2D case and one
+// for the 3D case. These classes contain the specific features of the test
+// cases, such as the inlet velocity and the initial conditions.
+
+// This is a template class for the flow past a cylinder test case. It is
+// templated on the dimension, which can be either 2 or 3. The reason for this
+// is that we need the dimension to be known at compile time. The class is
+// derived from the NavierStokes class, which is a template class itself, for
+// the same reason. A similar approach is used in most of our other classes.
+template <unsigned int dim>
+class Cylinder : public NavierStokes<dim> {
+ public:
+  // Virtual destructor.
+  virtual ~Cylinder() = default;
+
+  // Update lift and drag forces. In particular, this function computes the
+  // lift and drag coefficients and prints them to the console. These
+  // coefficients will be added to a .csv file and plotted in a Python script.
+  void update_lift_drag();
```

Comment:
It is generally advisable to separate the computations from the output. In this case, for instance, at some point in the future you may want to use the drag and lift coefficients e.g. in an optimal control loop.

Therefore, from a design perspective, it would probably be better to have this method return its result (e.g. as an `std::pair<double, double>`) instead of writing it to the standard output and/or to a CSV file. The output can be managed separately (e.g. in the output method itself).